### ISSUE TITLE
Certificat has been expired 

### ISSUE THREAD

**Issue by imflop** [SpectoLabs/hoverfly#1011]
## Description of the bug
https://github.com/SpectoLabs/hoverfly/commit/d9c53c0db80ae6cdca13e3ade8dda70d76387668
Today the day when the certificate can no longer be used.
### Observed result
**Hoverfly error messages seen (If none, say none)**
```
certificate verify failed: certificate has expired
```
**If possible, add screenshots to help explain your problem**

## Additional relevant information
1. Hoverfly version: 1.3.2
2. Anything that might help us to diagnose the problem

**Comment by tommysitu** [comment]
Hi @imflop, could you provide some details on how to reproduce this error please? thanks

**Comment by imflop** [comment]
Hi @tommysitu , sure:

```
openssl x509 -enddate -noout -in core/cert.pem
notAfter=Mar 23 13:46:27 2022 GMT
```

**Comment by tommysitu** [comment]
Hoverfly seems to work on my machine even it expires  🤷 . Anyway, it's due for renewal, I have extended it for another 10 years, should be fixed in 1.3.6 soon. Thanks for spotting it!
